src/libs/actions/Task.js

```diff
@@ -216,24 +219,29 @@ function completeTask(taskReportID, taskTitle, originalLastMessageText) {
     const failureData = [
         {
             onyxMethod: Onyx.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT}${taskReportID}`,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`,
             value: {
                 stateNum: CONST.REPORT.STATE_NUM.OPEN,
                 statusNum: CONST.REPORT.STATUS.OPEN,
```

Comment:
```suggestion
                stateNum: report.stateNum,
                statusNum: report.statusNum,
```
I guess we can do that, right?

Reply:
I think it makes sense (specially if we add other states eventually)! 

Changed that!